src/legacy/core_plugins/telemetry/public/services/telemetry_opt_in.ts

```diff
@@ -57,18 +60,58 @@ async function sendOptInStatus($injector: any, chrome: any, enabled: boolean) {
 }
 export function TelemetryOptInProvider($injector: any, chrome: any, sendOptInStatusChange = true) {
   currentOptInStatus = npStart.core.injectedMetadata.getInjectedVar('telemetryOptedIn') as boolean;
+
   const allowChangingOptInStatus = npStart.core.injectedMetadata.getInjectedVar(
     'allowChangingOptInStatus'
   ) as boolean;
 
+  telemetryNotifyUserAboutOptInDefault = npStart.core.injectedMetadata.getInjectedVar(
+    'telemetryNotifyUserAboutOptInDefault'
+  ) as boolean;
```

Comment:
I'm not so sure on this naming, usually boolean variables need an `is/should` prefix for clarity. 

Maybe `telemetryNotifyUserAboutOptInDefault `?

I think we want to keep the `telemetry` prefix because of the way the injectedVars are not namespaced by plugin